tasks/main.yml

```diff
@@ -145,9 +145,18 @@
   command: "{{ puppet_agent_bin_path }} agent --enable"
   when: puppetize_enable_puppet
 
+- name: Dry run (no changes, depending on the puppet code) of Puppet
+  command: "{{ puppet_agent_bin_path }} agent -t --noop"
+  register: puppet_result
```

Comment:
maybe we want to print out the result, if puppet is run in noop mode? Otherwise we need to ssh to each and every node to check what was the output from the log. Please note that output from Puppet might be printed out in a quite messy manner, it would be important that the newlines in the output are actually appearing as newlines in the printout

Reply:
Now it shows the result if the run doesn't finish without changes and errors (return code 0)